docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -27,7 +27,7 @@ This article assumes that you know how to use a command line and have a preferre
 1. Open a Command Line/Terminal.
```

Comment:
Made the change for this one instance.  The title is "Getting started with Command Line.." should we consider changing all instances of "command line" to "command prompt"